tests/console_test.go

```diff
@@ -108,35 +100,8 @@ var _ = Describe("[rfe_id:127][posneg:negative][crit:medium][vendor:cnv-qe@redha
 				})
 			})
 
-			Context("with an alpine image", func() {
-				type vmiBuilder func() (*v1.VirtualMachineInstance, *cdiv1.DataVolume)
-
-				newVirtualMachineInstanceWithAlpineContainerDisk := func() (*v1.VirtualMachineInstance, *cdiv1.DataVolume) {
-					return tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskAlpine)), nil
-				}
-
-				newVirtualMachineInstanceWithAlpineFileDisk := func() (*v1.VirtualMachineInstance, *cdiv1.DataVolume) {
-					return tests.NewRandomVirtualMachineInstanceWithFileDisk(cd.DataVolumeImportUrlForContainerDisk(cd.ContainerDiskAlpine), util.NamespaceTestDefault, k8sv1.ReadWriteOnce)
-				}
-
-				newVirtualMachineInstanceWithAlpineBlockDisk := func() (*v1.VirtualMachineInstance, *cdiv1.DataVolume) {
-					return tests.NewRandomVirtualMachineInstanceWithBlockDisk(cd.DataVolumeImportUrlForContainerDisk(cd.ContainerDiskAlpine), util.NamespaceTestDefault, k8sv1.ReadWriteOnce)
-				}
-
-				table.DescribeTable("should return that we are running alpine", func(createVMI vmiBuilder) {
-					vmi, dv := createVMI()
-					defer deleteDataVolume(dv)
-					RunVMIAndWaitForStart(vmi)
-					ExpectConsoleOutput(vmi, "login")
-				},
-					table.Entry("[test_id:4636]with ContainerDisk", newVirtualMachineInstanceWithAlpineContainerDisk),
```

Comment:
I am not sure it is correct to remove these tests so fast. At least I would ping someone responsible for these tests ids so they don't miss them.

Reply:
I agree, all else is good. But why removing the tests

Reply:
Do you see value in these tests?
I see that they were created long time ago by @ahadas's https://github.com/kubevirt/kubevirt/pull/2617 which @vladikr approved; do you recall why the backing storage is related to console?

test_id:4636 is inherently covered by test_id:1590
test_id:4637 is covered (also) by test_id:1520
test_id:4638 is covered by test_id:3139

I will drop this commit from this PR if you require; but the motivation for this PR is to simplify our tests here. It should be very clear why something is tested and why it is in this file.

Reply:
I didn't see immediate value and that's why I asked to have a deeper look.

Reply:
NOTE:
1) In "vm_test.go", The 3 tests (1520) are only checking whether the VM Status is Ready. 
2) In "console_test.go", Here we are also checking, access to the "Console" as these are console specific tests.
3) test_id: 4638 uses DV and test_id:3139 uses PVC, so two separate TestCases, I think.

If we want to clean up duplicate tests, we can probably remove only the below:
4) test_id: 4636 can probably be dropped, in favor of 1590, as we checking for console access almost 5 times.

Reply:
thanks @kbidarkar. You convinced me that there is an added value of the tests that stay here: they prove that the guest boots and becomes responsive with different storage back ends. I don't think that they belong here as they use the console to test bootability, they do not test console. I hope I can find a better place for them, but at the moment I'll keep them here.